client/views/room/lib/Toolbox/defaultActions.ts

```diff
@@ -15,7 +15,7 @@ addAction('rocket-search', {
 });
 
 addAction('user-info', {
-	groups: ['direct'],
+	groups: ['direct', 'live'],
 	id: 'user-info',
 	title: 'User_Info',
```

Comment:
```suggestion
	title: 'Contact_Info',
```

The button will open the `Contact Info`  panel, not `User Info`.